fleet/lib/kube-prometheus-stack/aggregator/values.yaml

```diff
@@ -0,0 +1,246 @@
+        proxy_ssl_verify off;
+    paths:
+      - /
+    pathType: Prefix
+    hosts:
+      - "prometheus.${ .ClusterName }.${ .ClusterLabels.site }.lsst.org"
+    tls:
+      - secretName: tls-prometheus-ingress
+        hosts:
+          - "prometheus.${ .ClusterName }.${ .ClusterLabels.site }.lsst.org"
+
+alertmanager:
+  alertmanagerSpec:
+    resources:
+      limits:
+        cpu: 1
+        memory: 512Mi
+      requests:
+        cpu: 100m
+        memory: 512Mi
+    externalUrl: "https://alertmanager.${ .ClusterName }.${ .ClusterLabels.site }.lsst.org"
+    secrets:
+      - o11y-webhooks
+  ingress:
+    enabled: true
+    annotations:
+      cert-manager.io/cluster-issuer: letsencrypt
+      kubernetes.io/ingress.class: nginx
+      nginx.ingress.kubernetes.io/backend-protocol: HTTP
+      nginx.ingress.kubernetes.io/server-snippet: |
+        proxy_ssl_verify off;
+    paths:
+      - /
+    pathType: Prefix
+    hosts:
+      - "alertmanager.${ .ClusterName }.${ .ClusterLabels.site }.lsst.org"
+    tls:
+      - secretName: tls-alertmanager-ingress
+        hosts:
+          - "alertmanager.${ .ClusterName }.${ .ClusterLabels.site }.lsst.org"
+  config:
+    global:
+      resolve_timeout: 5m
+      slack_api_url_file: /etc/alertmanager/secrets/o11y-webhooks/o11y-slack
+    route:
+      group_by: [namespace, cluster]
+      group_wait: 30s
+      group_interval: 5m
+      repeat_interval: 24h
+      receiver: slack-test
+      routes:
+        - receiver: "null"
+          matchers:
+            - alertname = "InfoInhibitor"
+        - receiver: watchdog
+          matchers:
+            - alertname = "Watchdog"
+        - receiver: squadcast-test
+          matchers:
+            - receiver =~ ".*,squadcast,.*"
+          continue: true
+    receivers:
+      - name: "null"
+      - name: watchdog
+      - name: slack-test
+        slack_configs:
+          - username: "${ .ClusterName }-prometheus"
+            channel: "#rubinobs-monitoring-test"
+            send_resolved: true
+      - name: squadcast-test
+        webhook_configs:
+          - url_file: /etc/alertmanger/secrets/o11y-webhooks/o11y-squadcast
+    inhibit_rules:
+      - source_matchers:
+          - alertname = "InfoInhibitor"
+        target_matchers:
+          - severity = "info"
+        equal: [namespace]
+      - source_matchers:
+          - severity = "critical"
+        target_matchers:
+          - severity =~ "info|warning"
+        equal: [alertname]
+      - source_matchers:
+          - severity = "warning"
+        target_matchers:
+          - severity = "info"
+        equal: [alertname]
+    templates:
+      - /etc/alertmanager/config/*.tmpl
+
+grafana:
+  resources:
+    limits:
+      cpu: 4
+      memory: 512Mi
+    requests:
+      cpu: 250m
+      memory: 512Mi
+  persistence:
+    enabled: true
+  deploymentStrategy:
+    type: Recreate  # default is RollingUpdate, which doesn't work w/ persistence enabled
+  grafana.ini:
```

Comment:
Currently the config is connected to LDAP, which I'm fine with. But the goal was moving towards Keycloack.

```
[auth.generic_oauth]
  enabled = true
  name = "<host>"
  allow_sign_up = true
  scopes = "profile,email,groups"
  auth_url = "https://<host>/auth/realms/master/protocol/openid-connect/auth"
  token_url = "https://<host>/auth/realms/master/protocol/openid-connect/token"
  api_url = "https://<host>/auth/realms/master/protocol/openid-connect/userinfo"
  signout_redirect_url = "https://<host>/auth/realms/master/protocol/openid-connect/logout?post_logout_redirect_uri=https%3A%2F%2Fgrafana.ayekan.dev.lsst.org%2Flogin"
  client_id = "$__file{/etc/secrets/keycloack-credentials/client_id}"
  client_secret = "$__file{/etc/secrets/keycloack-credentials/client_secret}"
  role_attribute_path = "contains(groups[*], 'grafana-admin') && 'GrafanaAdmin' || contains(groups[*], 'grafana-admin') && 'Admin' || contains(groups[*], 'grafana-editor') && 'Editor' || 'Viewer'"
  group_attribute_path = "groups"
  allow_assign_grafana_admin = true
```

Reply:
There was an existing deployment on pillan which used ldap (the summit grafana instance is also using ldap) that wasn't in this repo (which was done by someone who has since departed the team).  It was recreated as it was to get it up and running again several weeks ago.

When switching to keycloak is ready for a production deployment, please PR it.

Reply:
Then I'm just leaving it here for archival.